design-docs/improved-console.md

```diff
@@ -0,0 +1,283 @@
+# Improved Console Output
+
+## Primary Goals
+* Improve perceived performance by making user aware of background work
+* Modern feel
+* User extensibility
+
+### User Goals for Console
+In priority order, the user wants to answer the following questions:
```

Comment:
I don't think it's important to indicate a priority in the introduction. I would have just said that these are the goals with out a specific order. The priority of work items is already laid out by the stories below.